### ISSUE TITLE
[BUG] Streaming is not supported on this browser

### ISSUE THREAD

**Issue by KirotVieta** [hmlendea/geforcenow-electron#213]
### Describe the bug

Title says it all, I've updated UserAgent to latest, it happened after last geforce now update

### To Reproduce

Just launch the electron client

### Operating System

Ubuntu 22.04

### Desktop Environment

Ubuntu

### Display Server

Ubuntu

### Installation method

Desktop linux

### Version

Latest

### Is this a fresh install of the app or an update from a past version?

Fresh re-install

### Did this issue appear right away upon installation/updating, or spontaneously?

Yes, right after installation

**Comment by jdagdelen** [comment]
Any updates on this? Do you have any ideas on how the site is detecting it't not regular Chrome?

**Comment by KirotVieta** [comment]
According to DevTools, some feature that GFN checks for is Unsupported.

**Comment by jdagdelen** [comment]
Do you know what those are? I'd be interested in taking a crazy at it

**Comment by KirotVieta** [comment]
No, not exactly, however I know it happened after the H265 support update, which was released somewhere around the date I posted this issue.

**Comment by hmlendea** [comment]
Please re-open if this issue still affects the latest version of this app.
